python/torch_mlir/extras/onnx_importer.py

```diff
@@ -209,11 +285,23 @@ def define_function(
                 for out in graph_info.output_map.values()
             ]
             ftype = FunctionType.get(input_types, output_types)
-            func_op = func_dialect.FuncOp(func_name, ftype, ip=InsertionPoint(body))
+            func_op = func_dialect.FuncOp(
+                func_name,
+                ftype,
+                ip=InsertionPoint(body),
+                visibility="public" if public else "private",
```

Comment:
Lets avoid changing all imports with explicit public vs private. Keep `visibility` set to the same default before and set private in only the private cases. I looked at the testsuite run and this appears to have caused an uneeded change on all function signatures.

Reply:
I don't like the unnecessary change either, but the problem is that the default visibility is effectively private, and so these new functions don't get thrown away after inlining. At least, that's what happens with `torch-mlir-opt --inline`. Is there a better solution? Or, is it maybe acceptable that inlining won't clean things up? I don't know enough to answer these.

Reply:
We discussed this privately and it turns out that default visibility is actually like _public_, and so this can be avoided, yay. :)